godot-codegen/src/codegen_special_cases.rs

```diff
@@ -149,6 +149,7 @@ const SELECTED_CLASSES: &[&str] = &[
     "PathFollow2D",
     "PhysicsBody2D",
     "PrimitiveMesh",
+    "ProjectSettings",
```

Comment:
Is this really needed? Since we have an ongoing compile-time issue, I'd like to cut down this list rather than extend it.

I saw you added a test for the presence of this, but do you expect regressions on that end? Especially since the exclusion of `ProjectSettings` was just an oversight on my part and not a bug on its own.